libcst/codemod/_cli.py

```diff
@@ -373,35 +358,41 @@ def _parallel_exec_process_stub(  # noqa: C901
             newcode = ""
 
         # Inform success
-        result_queue.put(
-            ParallelExecResult(
-                filename=filename,
-                changed=changed,
-                transform_result=TransformSuccess(
-                    warning_messages=transformer.context.warnings, code=newcode
-                ),
-            )
+        return ExecutionResult(
+            filename=filename,
+            changed=changed,
+            transform_result=TransformSuccess(
+                warning_messages=transformer.context.warnings, code=newcode
+            ),
         )
     except KeyboardInterrupt:
-        result_queue.put(
-            ParallelExecResult(
-                filename=filename, changed=False, transform_result=TransformExit()
-            )
+        return ExecutionResult(
+            filename=filename, changed=False, transform_result=TransformExit()
         )
     except Exception as ex:
-        result_queue.put(
-            ParallelExecResult(
-                filename=filename,
-                changed=False,
-                transform_result=TransformFailure(
-                    error=ex,
-                    traceback_str=traceback.format_exc(),
-                    warning_messages=transformer.context.warnings,
-                ),
-            )
+        return ExecutionResult(
+            filename=filename,
+            changed=False,
+            transform_result=TransformFailure(
+                error=ex,
+                traceback_str=traceback.format_exc(),
+                warning_messages=transformer.context.warnings,
+            ),
         )
 
 
+# Unfortunate wrapper required since there is no `istarmap_unordered`...
+def _execute_transform_wrap(
+    args: Tuple[Codemod, str, ExecutionConfig],
+) -> ExecutionResult:
+    transformer, filename, config = args
+    return _execute_transform(
+        transformer=transformer,
+        filename=filename,
+        config=config,
+    )
```

Comment:
isn't this equivalent to the following?
```
lambda args: _execute_transform(*args)
```

Reply:
Currently yes, but in the case someone happens to reorder arguments or so, things would break in unexpected ways. (Explicit is better than implicit and all that.)

I'll change it to a dict so we can use `**kwargs`.